tinyfl/aggregator/__init__.py

```diff
@@ -110,10 +110,16 @@ async def handle(req: Request, background_tasks: BackgroundTasks):
         case Register(url=url):
             with client_lock:
                 clients.add(url)
+            logger.info("Client registered")
```

Comment:
log the client's url as well
```suggestion
            logger.info(f"Client {url} registered")
```